helper/builtinplugins/registry.go

```diff
@@ -91,6 +92,8 @@ func newRegistry() *registry {
 			"cassandra-database-plugin":  {Factory: dbCass.New},
 			"influxdb-database-plugin":   {Factory: dbInflux.New},
 			"postgresql-database-plugin": {Factory: dbPostgres.New},
+			"redis-database-plugin":      {Factory: dbValkey.New},
+			"valkey-database-plugin":     {Factory: dbValkey.New},
```

Comment:
```suggestion
			"valkey-database-plugin":     {Factory: dbValkey.New},
```

Reply:
@JanMa I think keeping this reference will allow anyone from Vault moving to OpenBao with existing Redis plugin support to continue using the new Valkey version transparently.